plugins/synapticsmst/fu-plugin-synapticsmst.c

```diff
@@ -151,7 +151,7 @@ fu_plugin_synaptics_add_device (FuPlugin *plugin,
 	dev_id_str = g_strdup_printf ("MST-%s-%s-%u-%u",
 				      kind_str, aux_node, layer, rad);
 	fu_device_set_id (dev, dev_id_str);
-	fu_device_set_platform_id (dev, aux_node);
+	fu_device_set_physical_id (dev, aux_node);
```

Comment:
I guess Synaptics doesn't break because it has both ID and physical ID set, let me see if it makes sense to split this up too and make logical ID for it.

Reply:
No, I don't think it makes sense to change synaptics.  It's got several levels of what I would say are "logical" IDs when it comes to supporting cascaded or direct.  I think we should keep it as is and let it do it's own IDs.

Reply:
Great, thanks. I do think this PR makes things a lot easier to understand.